FLEDGE.md

```diff
@@ -834,6 +863,35 @@ const maxAdComponents = navigator.protectedAudience ?
 
 The output of `generateBid()` can use the on-device ad composition flow through an optional adComponents field, listing additional URLs made available to the fenced frame the container URL is loaded in. The component URLs may be retrieved by calling `navigator.adAuctionComponents(numComponents)`, where numComponents will be capped to the maximum permitted value. To prevent bidder worklets from using this as a side channel to leak additional data to the fenced frame, exactly numComponents obfuscated URLs will be returned by this method, regardless of how many adComponent URLs were actually in the bid, even if the bid contained no adComponents, and the Interest Group itself had no adComponents either.
 
+##### 3.4.1 Flexible Component Ad Selection Considering k-anonymity
+
+Since a bid containing multiple components requires every component to be k-anonymous to
+display, it may be difficult to make such bids successfully. To make this easier,
+`generateBid()` can permit the browser to select a subset of returned component ads
+that pass the k-anonymity threshold.
+
+Note that feature detection for this in Chrome is the same as for multiple bid
```

Comment:
Is this feature detection mechanism proprietary? Or is it well-specified?

If it's the latter, please remove the "in Chrome" reference from the explainer.

Reply:
The way we have been spec'ing it as always returning constants:
https://wicg.github.io/turtledove/#feature-detection
The hope is that a full implementation will support all of it.

Perhaps if/when someone else implements it they'll find it helpful to not support it in need of incremental development as well, however.

Reply:
Rather https://wicg.github.io/turtledove/#feature-detection:~:text=Set%20browserSignals%5B%22multiBidLimit%22%5D%20to%20multiBidLimit. for this one